distributed/scheduler.py

```diff
@@ -3860,8 +3861,13 @@ async def start_unsafe(self):
 
         for listener in self.listeners:
             logger.info("  Scheduler at: %25s", listener.contact_address)
-        for k, v in self.services.items():
-            logger.info("%11s at: %25s", k, "%s:%d" % (listen_ip, v.port))
+        for name, server in self.services.items():
+            if name == "dashboard":
+                addr = listener.contact_address.split("://")[1].split(":")[0]
+                link = format_dashboard_link(addr, server.port)
+            else:
+                link = f"{listen_ip}:{server.port}"
+            logger.info("%11s at: %25s", name, link)
```

Comment:
Nit: it would be nice if the addresses were aligned around the `://`, currently they are one character off.

```
2023-03-10 15:56:55,683 - distributed.scheduler - INFO -   Scheduler at:   tcp://10.51.100.43:8786
2023-03-10 15:56:55,683 - distributed.scheduler - INFO -   dashboard at: http://10.51.100.43:8787/status
```

Perhaps they could be tweaked to line up.

```suggestion
            logger.info("%11s at:  %25s", name, link)
```